### DISCUSSION TITLE
Self hosted renovate stopped to update terraform-version

### DISCUSSION THREAD

**Discussion by martinachtner** [renovatebot/renovate#21031] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.73.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

7.17.13

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I want renovate to update my required terraform version to the latest available.

### What you tried so far.

I was able to see PRs for the terraform version created with renovate 32.72.0, when suing 32.73 or above, these PRs get autoclosed.
I looked at the changes for 32.73.0 and the docs, but that did not find any obvious configuration option that I might have missed.

### Relevant debug logs

<details><summary>Logs</summary>

```
 INFO: Repository started (repository=***)
       "renovateVersion": "32.72.0"
 INFO: Dependency extraction complete (repository=***)
       "baseBranch": "master",
       "stats": {
         "managers": {
           "terraform": {"fileCount": 40, "depCount": 138},
           "terraform-version": {"fileCount": 1, "depCount": 1}
         },
         "total": {"fileCount": 41, "depCount": 139}
       }
INFO: PR created (repository=***, branch=renovate/hashicorp-terraform-1.x)
       "prTitle": "RENOVATE: Update dependency hashicorp/terraform to v1.4.2"
.
.
.
INFO: Repository started (repository=***)
       "renovateVersion": "32.73.0"
 INFO: Dependency extraction complete (repository=***)
       "baseBranch": "master",
       "stats": {
         "managers": {
           "terraform": {"fileCount": 40, "depCount": 138},
           "terraform-version": {"fileCount": 1, "depCount": 1}
         },
         "total": {"fileCount": 41, "depCount": 139}
       }

INFO: Autoclosing PR (repository=***)
       "branchName": "renovate/hashicorp-terraform-1.x",
       "prTitle": "RENOVATE: Update dependency hashicorp/terraform to v1.4.2"
```

</details>

**Comment by martinachtner** [comment, marked as answer]
Update:
DEBUG: packageFiles with updates
"terraform-version": [
           {
             "packageFile": ".terraform-version",
             "deps": [
               {
                 "depName": "hashicorp/terraform",
                 "currentValue": "1.2.7",
                 "datasource": "github-releases",
                 "depIndex": 0,
                 "updates": [],
                 "warnings": [
                   {
                     "topic": "hashicorp/terraform",
                     "message": "Failed to look up dependency hashicorp/terraform"
                   }
                 ],
                 "versioning": "hashicorp"
               }
             ]
           }
         ]

**Reply by martinachtner** [reply]
I think I've found the issue:
From 32.73.0 onwards, there is a request to github's graphql API, which failed for me, because I did not supply a GITHUB_COM_TOKEN.
After trying it with the token, the behavor seems to be corrected.
Is this a bug or expected behavior?

**Reply by rarkins** [reply]
Expected
